spec/transaction/components/schemas.yaml

```diff
@@ -57,6 +57,13 @@ components:
                   title: Licence for the free geodata of the Federal Office of Topography swisstopo
                 - href: https://www.geocat.ch/geonetwork/srv/eng/catalog.search#/metadata/4c8c7c58-61c7-4a61-8e7a-6ffb95d183df
                   rel: describedby
+    collectionUpsertItems:
+      type: object
+      properties:
+        items:
```

Comment:
after discussion: rename to `features` since in the GET payload `features` is used as well (hence it's consistently confusing since in the path we have `../items`)